src/Spryker/Client/ProductLabelStorage/ProductLabelStorageClientInterface.php

```diff
@@ -51,14 +51,16 @@ public function findLabels(array $idProductLabels, $localeName);
 
     /**
      * Specification:
-     * - TODO: add specification
+     * - Retrieves label for given label name, locale and store name.
+     * - Forward compatibility (from the next major): only label assigned with passed $storeName will be returned.
```

Comment:
Need to clarify this part `(from the next major)` because technically next major do the same, only third major will do expected.

Reply:
I suggest to not use this part `(from the next major)` because  technically it shouldn't be major for this module.